scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/ServerJobAndTaskLogs.java

```diff
@@ -84,28 +84,26 @@ public static String getJobLog(JobId jobId, Set<TaskId> tasks) {
 
     }
 
-    public static void remove(JobId jobId, Set<TaskId> taskIds) {
-        removeLog(JobLogger.getJobLogFilename(jobId));
-        for (TaskId taskId : taskIds) {
-            removeLog(TaskLogger.getTaskLogFilename(taskId));
-        }
+    public static void remove(JobId jobId) {
+        removeFolderLog(jobId.value());
     }
 
-    private static void removeLog(String filename) {
-        for (String suffix : new String[] { "", ".1" }) {
-            removeFile(filename + suffix);
-        }
+    private static void removeFolderLog(String folderName) {
+        removeFile(folderName);
```

Comment:
if removeFile is not called somewhere else then it's better to merge the two methods and call it removeLogFolder